sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusProcessor.cs

```diff
@@ -1088,7 +1107,7 @@ private async Task ReconcileConcurrencyAsync()
             else if (diff < 0)
             {
                 var activeTasks = TaskTuples.Where(t => !t.Task.IsCompleted).ToList();
-                int excessTasks = activeTasks.Count - _maxConcurrentCalls;
+                int excessTasks = activeTasks.Count - maxConcurrentCalls;
```

Comment:
I was wondering if this is an oversight because the majority of the code here actually intends to use the copied value which wasn't the case here

Reply:
Yep, it sure was 😄 
Thanks for fixing!